eng/pipelines/runtime.yml

```diff
@@ -505,26 +505,8 @@ extends:
           jobParameters:
             testScope: innerloop
             nameSuffix: CoreCLR_TwoStage
-            buildArgs: -s clr+libs+host -c $(_BuildConfig) -rc Checked -p:StageOneBuild=true
+            buildArgs: -s clr+libs+host -c $(_BuildConfig) -rc Checked --bootstrap
```

Comment:
Would it make sense to default community platforms to bootstrap:true, perhaps under CrossBuild=true? Or do we want to keep it explicit?

Reply:
I think I'd rather have it explicit just because it makes it easier for me to know what command to run locally when reproing and makes less "spooky action at a distance" than if it were turned on automatically by `--cross` for some platforms and not others.